frontend/src/app/testResults/TestResultPrintModal.tsx

```diff
@@ -17,6 +17,12 @@ export const testQuery = gql`
     testResult(id: $id) {
       dateTested
       result
+      results {
```

Comment:
⚠️ I wanted to move this graphql query to the `frontend/src/app/testResults/operations.graphql` file but wasn't able to get the print modal to work when setting the query on `line 477` 😅 

Let me know if anyone has ideas on this or wants to pair to refactor this 😺